tests/meta/demultiplex/test_status_db_storage_functions.py

```diff
@@ -49,13 +52,16 @@ def test_update_sample_read_count(demultiplex_context: CGConfig):
     # GIVEN a sample and a read count
     sample = MagicMock()
     read_count = 100
+    sample.reads_updated_at = datetime.now()
```

Comment:
type strings in this function

Reply:
Type hinted the available variables, although the one mentioned is non-self.